core/store/src/main/java/uk/gov/gchq/gaffer/store/schema/ViewValidator.java

```diff
@@ -108,11 +112,15 @@ public ValidationResult validate(final View view, final Schema schema, final Set
                         }
 
                         result.add(validateAgainstStoreTraits(viewElDef, storeTraits));
-                        result.add(validateFunctionArgumentTypes(viewElDef.getPreAggregationFilter(), viewElDef, schemaElDef));
-                        result.add(validateFunctionArgumentTypes(viewElDef.getAggregator(), viewElDef, schemaElDef));
-                        result.add(validateFunctionArgumentTypes(viewElDef.getPostAggregationFilter(), viewElDef, schemaElDef));
-                        result.add(validateFunctionArgumentTypes(viewElDef.getTransformer(), viewElDef, schemaElDef));
-                        result.add(validateFunctionArgumentTypes(viewElDef.getPostTransformFilter(), viewElDef, schemaElDef));
+
+                        if (!view.hasBeenMigrated()) {
```

Comment:
Rather than calling it "hasBeenMigrated" perhaps it should be a config item called "skipValidation".